library/blob.h

```diff
@@ -415,6 +417,8 @@ struct eblob_backend {
 	struct list_head	bases;
 	int			max_index;
 
+	struct list_head	queue_to_sort;  // queue of closed bases to sort.
```

Comment:
The same way, I think that more preferable something like `closed_unsorted_bases` with no comment. Member name must be understandable.

Reply:
done